### PR TITLE
Update mdatp-portal-overview.png

### PR DESCRIPTION
**Pull request by msbemba**
Microsoft defender screenshot added

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7377

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/portal-overview.md [hunk 1]
```diff
@@ -41,10 +41,7 @@ You can use <a href="https://go.microsoft.com/fwlink/p/?linkid=2077139" target="
 
 ## Microsoft 365 Defender
 
-When you open the portal, you'll see:
-
-- (1) Navigation pane (select the horizontal lines at the top of the navigation pane to show or hide it)
-- (2) Search, Community center, Localization, Help and support, Feedback
+When you open the portal, you'll see Navigation pane (select the horizontal lines at the top of the navigation pane to show or hide it)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/portal-overview.md]:
```suggestion
When you open the portal, you'll see the Navigation pane (select the horizontal lines at the top of the navigation pane to show or hide it).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**